documentation/src/asciidoc/setting-up.adoc

```diff
@@ -152,3 +152,76 @@ class CalculatorSpec: Spek({
 IMPORTANT: As mentioned in the <<ide-support>> section, the IDEA plugin won't work if you're using the JUnit 4 runner.
 
 
+
```

Comment:
Maybe also ref this section instead of the legacy section?

`NOTE: For Android projects check <<setting-up-legacy>> section.`